internal/api/v1/admin/utxos/admin_utxo_filter_query_builder.go

```diff
@@ -0,0 +1,29 @@
+package utxos
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/querybuilders"
+	userutxos "github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/utxos"
```

Comment:
we are importing userutxos in adminutxos. Maybe it would be a good idea to separate that query builder into some common folder or something?

Reply:
Good point! I’ve been considering reorganizing the project tree structure to align more closely with endpoint domains. For example:

```
- api  
   - v1  
     - utxos  
       - query_builders  
       - user  
       - admin  
```

This kind of separation enhances cohesion and keeps the bounded context much cleaner. Additionally, the navigation part is much easier instead of jumping to another directories etc. 😸